src/components/Amalgamation/AmalgamatingBusinesses.vue

```diff
@@ -0,0 +1,255 @@
+<template>
+  <div id="amalgamating-businesses">
+    <v-btn
+      id="btn-add-amalgamating-business"
+      outlined
+      color="primary"
+      class="btn-outlined-primary"
+      :disabled="addAmalgatingBusinessDisabled"
+      @click="onAddBusinessClick()"
+    >
+      <v-icon>mdi-domain-plus</v-icon>
+      <span>Add an Amalgamating Business</span>
+    </v-btn>
+
+    <v-btn
+      v-if="isRoleStaff"
+      id="btn-add-amalgamating-foreign-business"
+      outlined
+      color="primary"
+      class="ml-2 btn-outlined-primary"
+      :disabled="addAmalgatingForeignBusinessDisabled"
+      @click="onAddForeignBusinessClick()"
+    >
+      <v-icon>mdi-domain-plus</v-icon>
+      <span>Add an Amalgamating Foreign Business</span>
+    </v-btn>
+
+    <!-- Add an Amalgamating Business button clicked -->
+    <v-expand-transition>
+      <v-card
+        v-if="isAddingAmalgamatingBusiness"
+        flat
+        class="section-container mt-4"
+      >
+        <v-row
+          no-gutters
+        >
+          <v-col
+            cols="12"
+            sm="3"
+          >
+            <label>Add a Business Registered in BC</label>
+          </v-col>
+
+          <v-col
+            cols="12"
+            sm="8"
+            class="ml-8"
+          >
+            <span>Enter the name or the incorporation number of the registered BC business
+              to add to this application.
+            </span>
+            <BusinessLookup
+              :showErrors="false"
+              :businessLookup="initialBusinessLookupObject"
+              :BusinessLookupServices="BusinessLookupServices"
+              label="Business Name or Incorporation Number"
+              @setBusiness="setAmalgamatingBusiness($event)"
```

Comment:
We want this business lookup to find only active businesses (not historical). Does it?

We want this business lookup to find only specific business types. Not FM I think. Probably not CP, either. Ask Mihai or Yui.

Reply:
Yes Sev, the business lookup only is searching for active businesses since that's the default value: https://github.com/bcgov/bcrs-shared-components/blob/2fc52fa9a8e4248a94f631909e01bb5ee0876d98/src/components/business-lookup/BusinessLookup.vue#L155

On it, will double check with Mihai or Yui for the second point.